tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/SpecValidationTool/SpecValidationTool.cs

```diff
@@ -75,6 +77,22 @@ public IList<string> RunTypeSpecValidation(string typeSpecProjectRootPath)
             }
         }
 
+        [McpServerTool(Name = "azsdk_check_typespec_project_in_public_repo"), Description("Check if TypeSpec project is in public spec repo. Provide absolute path to TypeSpec project root as param.")]
+        public string CheckTypeSpecProjectInPublicRepo(string typeSpecProjectPath)
```

Comment:
Can you put this MCP tool in a new class called SpecRepoValidationTool.cs? We are trying to change the structure where only one MCP tool resides in one class.